packages/apollo-server-core/src/requestPipeline.ts

```diff
@@ -17,7 +17,11 @@ import {
   enableGraphQLExtensions,
 } from 'graphql-extensions';
 import { DataSource } from 'apollo-datasource';
-import { PersistedQueryOptions } from '.';
+import {
+  PersistedQueryOptions,
+  symbolRequestListenerDispatcher,
+  enablePluginsForSchemaResolvers,
+} from '.';
```

Comment:
Huh, I've never seen `import {} ... from '.'` before. But I see it was pre-existing here... Is there a reason to not import directly from the particular file things come from?